admin/templates/common_auth/emails/invitation_email.html

```diff
@@ -0,0 +1 @@
+{% extends 'registration/password_reset_email.html' %}
```

Comment:
Perhaps the intention is to customize this form in future PRs (cf. https://github.com/CenterForOpenScience/osf.io/pull/4992#issuecomment-199289211)?